CommandLine/emake/OptionsParser.cpp

```diff
@@ -0,0 +1,505 @@
+#include "OptionsParser.hpp"
```

Comment:
This entire file can be replaced with a few lines of code atop [GFlags](https://gflags.github.io/gflags/), or my own [DeepFlags](https://github.com/JoshDreamland/DeepFlags/blob/master/DeepFlags.hpp), though I think the latter is overkill.